build-logic/documentation/src/main/groovy/gradlebuild/docs/dsl/source/GenerateApiMapping.java

```diff
@@ -30,13 +29,14 @@
 import org.gradle.api.tasks.PathSensitivity;
 import org.gradle.api.tasks.TaskAction;
 
+import javax.annotation.Nonnull;
 import java.io.FileWriter;
 import java.io.IOException;
 import java.io.PrintWriter;
 import java.util.Collection;
 import java.util.Map;
 
-@NonNullApi
+@Nonnull
```

Comment:
Any reason we're not using JSpecify for build logic as well?

Reply:
On Java code we can. Fixed.
On Groovy code we can't, see below.